src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,15 @@
-export const UserInfo = () => (
-  <>Put the user here</>
-);
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => {
+  const { name, email } = user;
+
+  return (
+    <p>
+      Posted by
+
```

Comment:
please add space before name
![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/91946036/c6923b5f-1330-434b-bb7d-4d2db2d3463b)